scripts/tests/ota_test.sh

```diff
@@ -3,17 +3,22 @@
 PASSCODE=${1:-20202021}
 DISCRIMINATOR=${2:-42}
 UDP_PORT=${3:-5560}
+OTA_DOWNLOAD_PATH=${4:-"/tmp/test.bin"}
+FIRMWARE_BIN="my-firmware.bin"
 
 pkill chip-ota-provider-app
 pkill chip-ota-requestor-app
+rm my-firmware.ota "$FIRMWARE_BIN" "$OTA_DOWNLOAD_PATH"
```

Comment:
nit: Might as well add one for the output file in case it ever changes in the future

```suggestion
rm "$FIRMWARE_OTA" "$FIRMWARE_BIN" "$OTA_DOWNLOAD_PATH"
```

Reply:
And add `-f` so the command doesn't fail if files don't exist.